monasca-log-agent/build.yml

```diff
@@ -0,0 +1,6 @@
+repository: monasca/log-agent
+variants:
+  - tag: master
+    aliases:
+      - :latest
+      - :master-{date}-{time}
```

Comment:
this should be in line 3, whereas this line should contain just master (I think the other images are doing it just like that).

Reply:
All other packages have the same as I did here

Reply:
Ok, my bad :rofl: